src/components/utilities/urls.js

```diff
@@ -34,6 +34,8 @@ const urls = {
 
   poLineView: id => `/orders/lines/view/${id}`,
 
+  viewInInventory: id => `/inventory/view/${id}`,
```

Comment:
Could probably change this to `viewInstance` instead?

Reply:
Done